conf/machine/include/imx-base.inc

```diff
@@ -372,6 +379,7 @@ IMX_EXTRA_FIRMWARE:mx8-generic-bsp   = "firmware-imx-8 imx-sc-firmware imx-seco"
 IMX_EXTRA_FIRMWARE:mx8m-generic-bsp  = "firmware-imx-8m"
 IMX_EXTRA_FIRMWARE:mx8x-generic-bsp  = "imx-sc-firmware imx-seco"
 IMX_EXTRA_FIRMWARE:mx8ulp-generic-bsp = "firmware-upower firmware-sentinel"
+IMX_EXTRA_FIRMWARE:mx9-nxp-bsp        = " firmware-imx-8 firmware-sentinel"
```

Comment:
firmware-imx-9 contains for DDR firmware for i.MX93. 
it is not right to use firmware-imx-8

Reply:
I have a patch on that recipe to install the right files. Let me push my latest